modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java

```diff
@@ -222,13 +225,18 @@ public int execute(List<String> rawArgs) {
 
         String commandName = "";
 
```

Comment:
Excess empty line.

Reply:
I think there should be an empty string here.